recipe/bld.bat

```diff
@@ -33,11 +33,11 @@ if errorlevel 1 exit 1
 if errorlevel 1 exit 1
 
 :: Install fix-up for a non version-specific boost include
-move %LIBRARY_INC%\boost-1_60\boost %LIBRARY_INC%
+move %LIBRARY_INC%\boost-1_61\boost %LIBRARY_INC%
 if errorlevel 1 exit 1
 
 :: Move dll's to LIBRARY_BIN
-move %LIBRARY_LIB%\*vc%VSTRING%0-mt-1_60.dll "%LIBRARY_BIN%"
+move %LIBRARY_LIB%\*vc%VSTRING%0-mt-1_61.dll "%LIBRARY_BIN%"
```

Comment:
Now that this appears to be a persistent feature, we should probably absorb it somehow. We probably can do something similar to [this]( https://github.com/conda-forge/tk-feedstock/blob/ceeb3993e637d82cbac4c131d6a477282c6ef2b3/recipe/bld.bat#L8-L9 ).